src/sdl_window.cpp

```diff
@@ -19,7 +29,414 @@
 #include <SDL3/SDL_metal.h>
 #endif
 
+Uint32 getMouseWheelEvent(const SDL_Event* event) {
+    if (event->type != SDL_EVENT_MOUSE_WHEEL)
+        return 0;
+    if (event->wheel.y > 0) {
+        return SDL_MOUSE_WHEEL_UP;
+    } else if (event->wheel.y < 0) {
+        return SDL_MOUSE_WHEEL_DOWN;
+    } else if (event->wheel.x > 0) {
+        return SDL_MOUSE_WHEEL_RIGHT;
+    } else if (event->wheel.x < 0) {
+        return SDL_MOUSE_WHEEL_LEFT;
+    }
+    return 0;
+}
+
+namespace KBMConfig {
+using Libraries::Pad::OrbisPadButtonDataOffset;
+
+// i wrapped it in a function so I can collapse it
+std::string getDefaultKeyboardConfig() {
```

Comment:
This needs to be a separate header, and a constexpr std::string_view so it has no runtime overhead